code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -22,11 +22,11 @@ makeMd = vcat . punctuate secSep . filtEmp
 introInfo :: String -> [String] -> Maybe String -> Doc
 introInfo name auths descr = introSec (text name) (listToDoc auths) (length auths) (maybePurpDoc descr)
 
--- | Instruction section, contains 3 paragraphs, Running, Building and Config Files.
+-- | Instruction section, contains 4 paragraphs, Running, Building, Input-Output and Config Files.
 -- The Config file section is only displayed if there are configuration files.
-instDoc :: [String] -> Doc
-instDoc cfp = regularSec (text "Making Examples") 
-    (runInstDoc <> doubleSep <> makeInstDoc) <> configSec cfp 
+instDoc :: [String] -> String -> Doc
+instDoc cfp name = regularSec (text "Making Examples") 
+    (runInstDoc <> doubleSep <> makeInstDoc <> doubleSep <> inOutFile name) <> configSec cfp 
```

Comment:
The instructions for changing input seem to be unconditional, i.e. input/output will always be done the same way (via files).

While this is true now, I don't think it will remain so. So, in a way, this code here makes an implicit assumption about the architecture of the generated code. And when we change that, it won't be obvious that this code needs to change as well.

So I'm unconfortable with the code as is.